lib/valueflow.cpp

```diff
@@ -4482,7 +4482,7 @@ static void valueFlowContainerAfterCondition(TokenList *tokenlist,
             vartok = vartok->tokAt(-3);
             if (!Token::Match(vartok, "%var% . %name% ("))
                 return cond;
-            if (!astIsContainer(vartok))
+            if (!astIsContainer(vartok) || !vartok->valueType()->container)
```

Comment:
I have the feeling this extra check can be added in `astIsContainer`, is there a reason to have it here?

Reply:
I would like to change it to fallback on the c++ names when the library is missing.